contracts/protocol/facets/Offer.sol

```diff
@@ -390,6 +393,9 @@ contract OfferFacet is Context, OfferErrors, Access, FundsLib, IOfferEvents {
             IFermionWrapper.unwrapToSelf,
             (_tokenId, exchangeToken, exchangeAmount)
         );
+
+        // Store the custom item price for later use in case of forceful fractionalisation
+        FermionStorage.protocolLookups().tokenLookups[_tokenId].itemPrice = customItemPrice;
```

Comment:
This value gets overwritten in 
https://github.com/fermionprotocol/contracts/blob/e429bf8ee2bbbdae9bd228137ba271de5ec4611a/contracts/protocol/facets/Offer.sol#L320

Probably the easiest fix is to change line 320 to
```solidity
if (_wrapType != FermionTypes.WrapType.SELF_SALE) tokenLookups.itemPrice = _priceDiscovery.price;
```
or something similar.